frontend/src/components/RequestTargetInputs.vue

```diff
@@ -72,6 +72,15 @@ const ignoreChains = computed(() =>
 );
 const { chainOptions } = useChainSelection(chains, ignoreChains);
 
+watch(
```

Comment:
There is only one example where i find this to be useful: 
user has no wallet connected -> selects a target chain -> selects the source chain to be the same as target chain.

All other cases are impossible in `production` environment due to the already implemented `ignoreChains` approach (check 5 lines above).

But this can also be solved by using the same `ignoreChains` approach (just like in this file) inside the `RequestSourceInputs.vue` component. That would be way more elegant and makes more sense. That way we can significantly reduce exposure to unpredicted errors due to unpredicted user behavior. :sweat_smile: 
If you implement that you can safely remove this piece of code too.

What do you think?

Reply:
Sorry, highlighted the wrong line with this one.

Reply:
We talked about this and will go with my approach.

The "no connected wallet" case is the only one which this is needed. The problem about the `ignoreChains` approach is there is an edge case where it is not safe with the current implementation. An unconnected user may select a target chain and the connect to the exact same chain which would select it in the source. This would be valid and the user would be able to create a request with the same source and target.